tests/integrations/raftstore/test_split_region.rs

```diff
@@ -270,7 +246,12 @@ impl Filter for EraseHeartbeatCommit {
     }
 }
 
-fn check_cluster(cluster: &mut Cluster<impl Simulator>, k: &[u8], v: &[u8], all_committed: bool) {
+fn check_cluster(
```

Comment:
Why this is only needed for v1?

Reply:
I changed this due to the change of the import, I will restore it. It is needed for v2 in some time. Now, the test using it cannot be passed due to some unkonwn reasons, so in this PR, it may not be used.